FAQs/README.md

```diff
@@ -25,6 +25,9 @@
 - ### Control Remediation
  1. [I want to turn OFF public access on key vault, but it is consumed by Azure Function Apps using consumption plan, How do I add the IP Addresses which are dynamically changing?](#1-i-want-to-turn-off-public-access-on-key-vault-but-it-is-consumed-by-azure-function-apps-using-consumption-plan-how-do-i-add-the-ip-addresses-which-are-dynamically-changing)
 
+ - ### UI
+ 1. [I am not able to see subscriptions in AzTS UI. Are there any prequisites or any settings that I am missing?](#1-i-am-not-able-to-see-subscriptions-in-azts-ui-are-there-any-prequisites-or-any-settings-that-i-am-missing)
```

Comment:
I can't see my subscriptions in the AzTS UI. What prerequisites or settings might I be missing?

Reply:
Updated